### ISSUE TITLE
Add Resource Manager Enablement to Project Resource

### ISSUE THREAD

**Issue by travisrandolph-bestbuy** [GoogleCloudPlatform/k8s-config-connector#382]
What we are proposing is adding the enablement of default services through the Project resource. This would be through an annotation. Default services for us would mean enabling the Resource Manager API. But there could be others. This would solve our issue for [364](https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/364) and potentially [362](https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/362).

```yaml
apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
kind: Project
metadata:
 annotations:
   cnrm.cloud.google.com/folder-id: "some-folder-id"
   cnrm.cloud.google.com/default-api: "true"
 name: test-project
spec:
 name: test-project
 billingAccountRef:
   external: "some-billing-id"
```

**Comment by maqiuyujoyce** [comment]
Hi @travisrandolph-bestbuy , thank you for your suggestions.

For #364 , you should be able to manage `Service` resources of the same API for different projects within the same namespace [using the `resourceID` field](https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/364#issuecomment-770021982).

For #362, because the customer created their projects via gcloud and Cloud Console (not Config Connector), I think enabling default services doesn't seem to be helpful in solving their problem.

Do you have any other scenario that services cannot be enabled properly with `Service` resources? Would like to learn more about it.

**Comment by travisrandolph-bestbuy** [comment]
Closing as what @maqiuyujoyce mention fixes #364.
